contrib/spec/podman.spec.in

```diff
@@ -52,7 +52,10 @@ ExclusiveArch: aarch64 %{arm} ppc64le s390x x86_64
 # The COPR process will uncomment this
 #BuildRequires: golang-bin
 #
+# btrfs-progs-devel package is not available in CentOS/RHEL-8
+%if 0%{?fedora} > 28 || 0%{?rhel} == 7 || 0%{?centos} == 7
```

Comment:
This still eliminates SUSE from using BTRFS 
Just do 
 %if 0%{?rhel} != 8 && 0%{?centos} != 8

Reply:
Thanks, Done :-)